python/tskit/tables.py

```diff
@@ -3379,6 +3380,16 @@ def simplify(
         The returned node mapping is always the identity mapping, such that
         ``a[u] == u`` for all nodes.
 
+        By default, the node sample flags are updated by unsetting
+        :data:`NODE_IS_SAMPLE` flag for all nodes and subsequently setting it
+        for the nodes provided as input to this function. Setting the
+        ``update_sample_flags`` parameter to False will prevent this from
+        occuring, making it the responsibility of calling code to keep track of
+        the ultimate sample status of nodes. This is mostly of use when
+        combined with the ``filter_nodes=False`` option, which will then
+        guarantee that the node table will not be changed in any way by
+        this operation.
```

Comment:
In the C docstring, it says that the absence of `filter_individuals` is needed to guarantee the nodes are untouched, should that be mentioned here?

Reply:
Good call, will fix.

Reply:
Presumably this applies to `filter_populations` too? I guess we can ignore migrations for the time being due to https://github.com/tskit-dev/tskit/issues/20, but do we need to think about what happens if we implement that?

Reply:
Fixed docs to mention pops and individuals. I don't think migrations can affect this, but it would be a case of updating the documentation here to note what affect it would have on the node table if/when we do support them.

Reply:
Yep - I'll mention that in #20 so we don't lose track.